### DISCUSSION TITLE
Hello team, some extra ideas/suggestions for you to evaluate

### DISCUSSION THREAD

**Discussion by StVMartin** [cpvalente/ontime#1316] [Ideas]
Hi guys, 

I've been using OT for some time now and must say once you get the hang of it it really is a great product.  Some elements I don't eally grasp their use, like the auxiliairy timer?  Is there any information on that?  I guess it is mainly a stage speaker timer?  with messaging functionality (which is great).

Some things that I am missing: 

1) a production contact page, which can be accessed via the server name, function, company titel, mobile, info / very often people are looking during a stress moment for the people in charge of specifics, like this the showcaller can decide on that communication flow easliy and more efficiently. 

2) A green room screen like the public and backstage, honestly I don't really get the difference between both, but I prefer using the backstage also for specific stage management and motion logistics, the public gives a general overview to clients etc.  But a green room view could say what happens on stage > time remaining till next > who should be ready (scene +1) > who should be preparing (scene +2)

3) I keep on having issues with the lack of timer within an event, let me elaborate, I would like to start a show sequence of let's say 10 minutes on a timecode file in one cue, then be able to run all specific cues within this event as timecode lines, where we can enter ourselves the starting timecode and it will count, the same pinciple as the event timers.  if you need to make a line per event, which mostly will be preprogrammed in timecode programs, the document becomes to large and chaotic, some cues only have like 2 seconds between them or even less.  So having the uption to use an event as a chapter for such a scene and below a excel like overview, timecode start > duration > Q > action would be a real asset I think. These could in the operator or backstage view appear as a simple table below the event if their view is activated. 

4) I don't get the use for the lowerr thirds, maybe some info on that?

5) I can not find the use of the countdown, maybe some info on that?

6) Can OT be connected to Streamdeck?

7) Move several lines together?  for those times people decide to change the show order?  Is this possible?

8) i can not seem to copy outside OT and paste inside OT?

9) An optimized view for stage managers on their tablets or mobile phones seems handy also, different layout for one (fixed) table of fields for stage management only? 

10) for programming purposes I would find it interesting to have the option to 'attach' a new field like INFO LIGHT below (not to the side) of the light field, so that the same document can be used for programming and preparation and simply hidden when in live show.   Now i use an extra column and create a field which makes the ROS extremely wide, especially when sharing, linked to google with external viewers. 

11) also, any way to simplify the way of connecting to google sheets or a file on dropbox, OneDrive...?

12) also last times when i exported to a linked google sheet i did not export any of the custom fields, what happens here?  My error or a bug?

13) A function 'renumber all cues) could also be very usefull

14) Maybe interesting for others: I personally during the live shows use a 'short cue' approach, only showing what I need to call, everybody knows what to do from rehearsals.  As some technical  cues often repeat, this could be a dropdown field, followed by a text field > example SVO (dropdown) Name speaker (text field)? Now I create these in different custom fields but again, it would be usefull that these appear below the cue title and note rather than next to it.  Good idea?  For you guys to judge.

And then another general question for anyone reading, I am looking for a lingo overview f cue names, LXQ etc.  as I notice that English showcallers use a different language (also musical running shows) then events people (and open air shows).  Would like to master all approaches and be able to adapt easily to service the team's habits. Anyone has a site elaborating on this?  Thanks,

StV

**Comment by alex-Arc** [comment]
Hey, I will start with some of the easy stuff 

4 - The lower third page is mostly there for legacy reasons and it still hase some users but we recommend that it is handled in a separate program (obs, vmix, CasparCG) and it likely won't get any further development 

5 - Yes use it with [companion](https://bitfocus.io/companion)

11 - The Google login process is as good as we can get without us having to pay Google for the auth request 
   How do you imagine a dropbox/onedrive 
Sync would work?

**Reply by StVMartin** [reply]
Hello Alex, 
4- clear
5- will try that

11- Basically what I would prefer is a one way feed, so that clients, partners have a read functionality via online link or via for instance a dropbox export.  Either I use the authentication and share the Google docs, but clients are so used to editing comments which obviouslt I do not want in a live rundown.  But it creates extra work for me to check all fields for remarks. 

Dropbox could be a simple pdf or excel export. Users link to their dropbox account (I think db allows easy access without cost), dont know how onedrive works in this aspect.  Ideally the export allows to choose which fields to export, so I could do one technical for all people wanting info upfront and only main info and timing for non technicals.

**Comment by cpvalente** [comment]
Hi @StVMartin,

I appreciate it took some time and effort for you to give us this feedback

I wanted to take some time to give you some proper answers. There is a lot to unpack on your suggestion so I thought that I could give it a short comment and classification in a similar way to what I would do if we were considering a new feature. Hope this is helpful and makes the process transparent
Keep in mind these are some initial thoughts. I may be convinced otherwise when we discuss this or as we go along with the roadmap.

Few notes:
- Cost of implementation relates to how much effort we need to create and maintain the feature
- User value relates to how much of the user base do we believe can benefit from the feature, and what overall value it brings to those users

I can also say that we follow a principle of progressive disclosure of complexity. This is a fancy way of saying that we want the users to easily onboard to the app and start using it, but slowly uncover the power that Ontime brings as they get familiar with it. This is the short answer for some of your questions, longer answers can be found [in our docs](https://docs.getontime.no/)

> Some elements I don't eally grasp their use, like the auxiliairy timer? Is there any information on that? I guess it is mainly a stage speaker timer? with messaging functionality (which is great).

The Auxiliary timer is, fundamentally, a timer which is disconnected from the schedule. In short, we dont want to prescribe what you do with the features, and you are welcome to ignore this until it makes sense to you.


### 1. a production contact page

I am not sure what this means for Ontime, I fear that it may provide an expectation of Ontime becoming a fully fledged management system. However, I believe that a minimal implementation could provide sufficient value

**Cost of implementation:** 2/5
**User value**: 3/5

### 2. A green room screen like the public and backstage

We would need a design proposal to create a new screen. I am unconvinced that you are proposing a new use case, but rather that you suggest that the current views do not provide sufficient information. In that case I would prefer a revision to the current design.

As to your question
> honestly I don't really get the difference between both

Public screen only shows events which are marked as `public`. I see this detail is missing from the docs and will update

### 3. having the uption to use an event as a chapter

Event groups have been discussed several times and is likely to be our next large effort. However, be advised that the specific use case of have multiple running events at the same time may not be the outcome. We have a ticker #204 which we will use to progress this feature

### 7. Move several lines together? 

No, this is currently not possible. We imagine that this would become a feature of event groups

**Cost of implementation:** 3/5
**User value**: 2/5

### 8. i can not seem to copy outside OT and paste inside OT?

It is unclear what you want to copy and paste? I can answer provided some more feedback

### 9. An optimized view for stage managers on their tablets or mobile phones

We are currently revising the designs to try and bridge this. However, we are short on resources to follow it through. We welcome sponsorship for design resources.

**Cost of implementation:** 4/5
**User value**: 3/5

### 10. have the option to 'attach' a new field like INFO LIGHT below (not to the side) of the light field

Some concept of subfields have been floated around. I understand the convenience but the complexity is too high for the value I see

**Cost of implementation:** 4/5
**User value**: 1/5

### 11. simplify the way of connecting to google sheets or a file on dropbox, OneDrive

We would need a use case to consider. As of now, I believe the file system management is simple enough that users can leverage. Extra steps could be extremely costly and provide little value

**Cost of implementation:** 4/5
**User value**: 1/5

### 12. last times when i exported to a linked google sheet i did not export any of the custom fields

Could you please open a bug ticket with information to recreate? The feature will write back the information it read. So if you initially read the custom fields from the google sheet import, it is expected that the export will work

### 13. A function 'renumber all cues) could also be very usefull

**Cost of implementation:** 3/5
**User value**: 2/5

### 14. 'short cue' approach

It sounds interesting, but I am unsure I have a good picture of your suggestion
Could you please provide more information here, perhaps with a screenshot of an example implementation?

Hope this helps

> Hi guys,
> 
> I've been using OT for some time now and must say once you get the hang of it it really is a great product. Some elements I don't eally grasp their use, like the auxiliairy timer? Is there any information on that? I guess it is mainly a stage speaker timer? with messaging functionality (which is great).
> 
> Some things that I am missing:
> 
> 1. a production contact page, which can be accessed via the server name, function, company titel, mobile, info / very often people are looking during a stress moment for the people in charge of specifics, like this the showcaller can decide on that communication flow easliy and more efficiently.
> 2. A green room screen like the public and backstage, honestly I don't really get the difference between both, but I prefer using the backstage also for specific stage management and motion logistics, the public gives a general overview to clients etc.  But a green room view could say what happens on stage > time remaining till next > who should be ready (scene +1) > who should be preparing (scene +2)
> 3. I keep on having issues with the lack of timer within an event, let me elaborate, I would like to start a show sequence of let's say 10 minutes on a timecode file in one cue, then be able to run all specific cues within this event as timecode lines, where we can enter ourselves the starting timecode and it will count, the same pinciple as the event timers.  if you need to make a line per event, which mostly will be preprogrammed in timecode programs, the document becomes to large and chaotic, some cues only have like 2 seconds between them or even less.  So having the uption to use an event as a chapter for such a scene and below a excel like overview, timecode start > duration > Q > action would be a real asset I think. These could in the operator or backstage view appear as a simple table below the event if their view is activated.
> 4. I don't get the use for the lowerr thirds, maybe some info on that?
> 5. I can not find the use of the countdown, maybe some info on that?
> 6. Can OT be connected to Streamdeck?
> 7. Move several lines together?  for those times people decide to change the show order?  Is this possible?
> 8. i can not seem to copy outside OT and paste inside OT?
> 9. An optimized view for stage managers on their tablets or mobile phones seems handy also, different layout for one (fixed) table of fields for stage management only?
> 10. for programming purposes I would find it interesting to have the option to 'attach' a new field like INFO LIGHT below (not to the side) of the light field, so that the same document can be used for programming and preparation and simply hidden when in live show.   Now i use an extra column and create a field which makes the ROS extremely wide, especially when sharing, linked to google with external viewers.
> 11. also, any way to simplify the way of connecting to google sheets or a file on dropbox, OneDrive...?
> 12. also last times when i exported to a linked google sheet i did not export any of the custom fields, what happens here?  My error or a bug?
> 13. A function 'renumber all cues) could also be very usefull
> 14. Maybe interesting for others: I personally during the live shows use a 'short cue' approach, only showing what I need to call, everybody knows what to do from rehearsals.  As some technical  cues often repeat, this could be a dropdown field, followed by a text field > example SVO (dropdown) Name speaker (text field)? Now I create these in different custom fields but again, it would be usefull that these appear below the cue title and note rather than next to it.  Good idea?  For you guys to judge.
> 
> And then another general question for anyone reading, I am looking for a lingo overview f cue names, LXQ etc. as I notice that English showcallers use a different language (also musical running shows) then events people (and open air shows). Would like to master all approaches and be able to adapt easily to service the team's habits. Anyone has a site elaborating on this? Thanks,
> 
> StV

**Reply by StVMartin** [reply]
**"I appreciate it took some time and effort for you to give us this feedback"** >>> my pleasure

**"1. a production contact page: I am not sure what this means for Ontime, I fear that it may provide an expectation of Ontime becoming a fully fledged management system. However, I believe that a minimal implementation could provide sufficient value"**
>>> for me this would ONLY be a contacvt page of show caller, head of production, maybe tech of venue, the 4 or 5 contacts that people may urgently need in case of technical worries, far from a full management system, quite the contrary please."

**"2. A green room screen like the public and backstage.  We would need a design proposal to create a new screen. I am unconvinced that you are proposing a new use case, but rather that you suggest that the current views do not provide sufficient information. In that case I would prefer a revision to the current design."** >>> currenly I used the public view with info to share on public places, audience visible and select only relevant fields.  The backstage could be used merely for stage managers and specific attention points, props on/off etc.  But a green room view could be identical from techincal point of view, but could then be used in the make-up, crew catering etc.  I use backstage now for this which works, but soo much info that is not relevant to the artist that they don't pay attention.  Basically this would show "what is NOW on, what is up next and should be ready, who should get ready.  Technically this can be exactly the same module as backstage or public."

**"3. having the option to use an event as a chapter: Event groups have been discussed several times and is likely to be our next large effort. However, be advised that the specific use case of have multiple running events at the same time may not be the outcome. We have a ticker https://github.com/cpvalente/ontime/issues/204 which we will use to progress this feature"** >>> if you need a tester, feel free to ask me.

**8. i can not seem to copy outside OT and paste inside OT?: It is unclear what you want to copy and paste? I can answer provided some more feedback"** >>> Mac M1 Max OS Sonoma here and when I for instance export to Google copy the ID from the url i can not use any past function to paste it into OT.  Same for copying a text field from an excel document let's say and pasting it into a field in OT, it does not work.  When sliding it over onto the field it does work.

14. 'short cue' approach: It sounds interesting, but I am unsure I have a good picture of your suggestion
Could you please provide more information here, perhaps with a screenshot of an example implementation? >>> "basically I use the different field columns in the cuesheet to allow techs to program and later on see their specific info which they may even have added, however I call a cue shortcut name like "SVO Speaker" which then implies all cues preprogrammed at that line.  In a way a text variation of the LXQ and number way of work which I find ok for long running shows, but which is too abstract for a one of show for techs to immediately be able to follow, also often we don't have enough progging an rehearsal time to check and double check all Q numbers to work in that way."

Hope this clarifies a bit?   Will also open a ticket for the export bug.
Thanks

StV
